custom-attachments/src/main/java/io/getstream/chat/android/customattachments/factory/DateAttachmentPreviewFactory.kt

```diff
@@ -0,0 +1,31 @@
+package io.getstream.chat.android.customattachments.factory
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import io.getstream.chat.android.client.models.Attachment
+import io.getstream.chat.android.customattachments.databinding.ItemDateAttachmentPreviewBinding
+import io.getstream.chat.android.ui.message.input.attachment.selected.internal.BaseSelectedCustomAttachmentViewHolder
+import io.getstream.chat.android.ui.message.input.attachment.selected.internal.SelectedCustomAttachmentViewHolderFactory
+
+class DateAttachmentPreviewFactory : SelectedCustomAttachmentViewHolderFactory {
+    override fun createAttachmentViewHolder(
```

Comment:
Really minor formatting nitpick:

I usually like having an empty line here, I think it makes things more "scannable".